packages/shared/src/contexts/payment/StoreKit.tsx

```diff
@@ -79,63 +137,29 @@ export const StoreKitSubProvider = ({
   const { user, isValidRegion: isPlusAvailable } = useAuthContext();
   const { logSubscriptionEvent } = usePlusSubscription();
   const { growthbook } = useGrowthBookContext();
-  const productIds = useFeature(featureIAPProducts);
-  const productList = useMemo(() => Object.keys(productIds), [productIds]);
   const logRef = useRef<typeof logSubscriptionEvent>();
   logRef.current = logSubscriptionEvent;
 
+  const { data } = useQuery<ProductPricingMetadata[]>({
```

Comment:
Maybe rename `data` to something more meaningful

Reply:
Done! 🚀